MSAL/src/configuration/MSALWebviewParameters.m

```diff
@@ -68,9 +62,7 @@ - (instancetype)initWithAuthPresentationViewController:(MSALViewController *)par
 
 - (id)copyWithZone:(__unused NSZone *)zone
 {
-    MSALWebviewParameters *item;
-    item = [[MSALWebviewParameters alloc] initWithAuthPresentationViewController:_parentViewController];
-    item.parentViewController = _parentViewController;
+    MSALWebviewParameters *item = [[MSALWebviewParameters alloc] initWithAuthPresentationViewController:_parentViewController];
```

Comment:
Looks like this line is missing in the new code:
`item.parentViewController = _parentViewController;`

Reply:
it is not needed, the `parentViewController ` is provided in the init method.